src/clue.js

```diff
@@ -1,14 +1,142 @@
 // ITERATION 1
 
+
 // Suspects Collection
-const suspectsArray = [];
+
+const suspectsArray = [ 
+{
+nickname: "mrGreen",
+firstName: "Jacobç",
+lastName: "Green",
+occupation: "Entrepreneur",
+age: "45", 
+description: "He has a lot of connections",
+image: "https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg", 
+color: "green"
+},
+
+{
+nickname: "drOrchid",
+firstName: "Doctor",
+lastName: "Orchid",
+occupation: "Scientist",
+age: "26",
+description: "PhD in plant toxicology. Adopted daughter of Mr. Boddy"
+image: "http://www.radiotimes.com/uploads/images/Original/111967.jpg",
+color: "white"
+},
+
+{
+nickname: "profPlum",
+firstName: "Victor",
+lastName: "Plum",
+occupation: "Designer",
+age: "22",
+description: "Billionaire video game designer",
+image: "https://66.media.tumblr.com/ee7155882178f73b3781603f0908617c/tumblr_phhxc7EhPJ1w5fh03_540.jpg",
+color: "purple"
+},
+
+{
+nickname: "missScarlet",
+firstName: "Kasandra",
+lastName: "Scarlet",
+occupation: "Actor",
+age: "31",
+description: "She is an A-list movie star with a dark past",
+image: "https://www.radiotimes.com/uploads/images/Original/111967.jpg",
+color: red
```

Comment:
So the main cause for `pickMystery` failing the tests is this here. The value `red` needed to be made into a string `"red"`
```suggestion
color: "red"
```